docs/buildkitd.toml.md

```diff
@@ -112,4 +112,8 @@ insecure-entitlements = [ "network.host", "security.insecure" ]
 # optionally mirror configuration can be done by defining it as a registry.
 [registry."yourmirror.local:5000"]
   http = true
+
+[log]
```

Comment:
Place this section right before `[grpc]`

Reply:
Done